src/pimp_mixer.c

```diff
@@ -51,21 +55,22 @@ PURE int pimp_calc_loop_event(int event_cursor, int event_delta, const int max_s
 		int denom = event_delta;
 		
 		/* call BIOS-function to divide, as it's fast enough for our needs. */
-		asm (
-			"mov r0, %[number] \n"
-			"mov r1, %[denom]  \n"
-#ifdef __thumb__
-			"swi 0x6           \n"
-#else
-			"swi 0x6 << 16     \n"
-#endif
-			"mov %[result], r0 \n"
-			: [result] "=r" (result) /* output */
-			: /* inputs */
-				[number] "r" (number),
-				[denom] "r" (denom)
-			: "r0", "r1", "r2", "r3" /* clobbers */
-		);
+		result = Div(number, denom);
+//		asm (
+//			"mov r0, %[number] \n"
+//			"mov r1, %[denom]  \n"
+//#ifdef __thumb__
+//			"swi 0x6           \n"
+//#else
+//			"swi 0x6 << 16     \n"
+//#endif
+//			"mov %[result], r0 \n"
+//			: [result] "=r" (result) /* output */
+//			: /* inputs */
+//				[number] "r" (number),
+//				[denom] "r" (denom)
+//			: "r0", "r1", "r2", "r3" /* clobbers */
+//		);
```

Comment:
So, I've tried not to use libgba in the player itself, only in the example. This is so people who doesn't use libgba can also use pimpmobile.

Reply:
See [above comment](https://www.pouet.net/prod.php?which=84533)